server/plugin.go

```diff
@@ -340,6 +343,30 @@ func (p *Plugin) handleList(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (p *Plugin) handleCount(w http.ResponseWriter, r *http.Request) {
+	userID := r.Header.Get("Mattermost-User-ID")
+
+	countIssues, err := p.listManager.CountIssues(userID)
+	if err != nil {
+		p.API.LogError("Unable to get issues for user err=" + err.Error())
+		p.handleErrorWithCode(w, http.StatusInternalServerError, "Unable to get issues for user", err)
+		return
+	}
+
+	countIssuesJSON, err := json.Marshal(countIssues)
+	if err != nil {
+		p.API.LogError("Unable marhsal count issue list to json err=" + err.Error())
+		p.handleErrorWithCode(w, http.StatusInternalServerError, "Unable marhsal count issue list to json", err)
```

Comment:
Nit: to de-duplicate some string creation here. Same with the block above this one

```suggestion
		msg := "Unable marshal count issue list to json"
		p.API.LogError(msg, "err", err.Error())
		p.handleErrorWithCode(w, http.StatusInternalServerError, msg, err)
```

Reply:
I see now that this is copied from the function above. I'm still thinking we can make this change here

Reply:
I just modified all the error messages the same way

Reply:
@raghavaggarwal2308 That sounds fine but we would want to change every call to `handleErrorWithCode` as well to avoid double logging. And that's definitely out of scope of this PR